extensions/extensions-docker/aissemble-hive/aissemble-hive-service/src/main/resources/docker/Dockerfile

```diff
@@ -15,8 +15,8 @@ ENV HIVE_HOME=/opt/hive
 COPY --from=appsource /opt/hadoop $HADOOP_HOME
 COPY --from=appsource /opt/hive $HIVE_HOME
 
```

Comment:
I: adding force flags to not throw an error on the existing group or user.  These have existed for some time, so I suspect this is an upstream Linux flavor update that made the underlying commands more sensitive.